### PR TITLE
Url does not update when changing product attributes 

### PR DESCRIPTION
**Pull request by PierreRambaud**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.4.x
| Description?  | Switching combination doesn't change the url.
| Type?         | bug fix
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/BOOM-5929
| How to test?  | Try switching color/size and refresh page

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9335)

### CODE REVIEW

File: themes/_core/js/product.js [hunk 1]
```diff
@@ -311,4 +311,17 @@ $(document).ready(() => {
...
+
+    prestashop.on('updatedProduct', (args) => {
+        if (!args.product_url) {
+            return;
+        }
+        window.history.pushState(
+            {
+              id_product_attribute: args.id_product_attribute
+            },
+            undefined,
```
Comment (eternoendless) [inline themes/_core/js/product.js]: Shouldn't this be an empty string?
https://developer.mozilla.org/en-US/docs/Web/API/History_API#The_pushState()_method

Comment (PierreRambaud) [reply themes/_core/js/product.js]: I just took what was on the previous prestashop version and from doc:`a title (which is currently ignored), `

Comment (PierreRambaud) [reply themes/_core/js/product.js]: I changed it to use `document.title` like in `admin-theme.js`

File: themes/_core/js/product.js [hunk 2]
```diff
@@ -311,4 +311,17 @@ $(document).ready(() => {
             }
         });
     });
+
+    prestashop.on('updatedProduct', (args) => {
+        if (!args.product_url) {
```
Comment (Quetzacoalt91) [inline themes/_core/js/product.js]: Don't you want to check `args.id_product_attribute` as well?

Comment (PierreRambaud) [reply themes/_core/js/product.js]: Done! Thanks

Comment (PierreRambaud) [reply themes/_core/js/product.js]: @mickaelandrieu One more time, double review was needed :trollface:

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by Quetzacoalt91** [comment]
Thank you @PierreRambaud

**Pull request merged**